cmd_actuators_mux/include/cmd_actuators_mux/cmd_actuator_mux_nodelet.h

```diff
@@ -0,0 +1,190 @@
+ *  are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above
+ *     copyright notice, this list of conditions and the following
+ *     disclaimer in the documentation and/or other materials provided
+ *     with the distribution.
+ *   * Neither the name of QinetiQ nor the names of its
+ *     contributors may be used to endorse or promote products derived
+ *     from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ *  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ *  FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *  COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ *  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ *  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ *  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *  POSSIBILITY OF SUCH DAMAGE.
+ *********************************************************************/
+
+#ifndef CMD_ACTUATORS_MUX_CMD_ACTUATOR_MUX_NODELET_H
+#define CMD_ACTUATORS_MUX_CMD_ACTUATOR_MUX_NODELET_H
+
+#include <dynamic_reconfigure/server.h>
+#include <nodelet/nodelet.h>
+#include <ros/ros.h>
+
+#include <string>
+
+#include "cmd_actuators_mux/cmd_actuator_subscribers.h"
+#include "cmd_actuators_mux/reloadConfig.h"
+#include "fin_control/SetAngles.h"
+#include "thruster_control/SetRPM.h"
+#include "std_msgs/String.h"
+
+namespace cmd_actuator_mux
+{
+class CmdActuatorMuxNodelet : public nodelet::Nodelet
+{
+ public:
+  virtual void onInit();
+
+  CmdActuatorMuxNodelet()
+  {
+    cmd_fin_angles_subs.allowed = VACANT;
+    cmd_set_rpm_subs.allowed = VACANT;
+
+    dynamic_reconfigure_server = NULL;
+  }
+
+  ~CmdActuatorMuxNodelet()
+  {
+    if (dynamic_reconfigure_server != NULL) delete dynamic_reconfigure_server;
+  }
+
+ private:
+  static const unsigned int VACANT =
+      666666; /**< ID for "nobody" active input; anything big is ok */
+  static const unsigned int GLOBAL_TIMER =
+      888888; /**< ID for the global timer functor; anything big is ok */
+
+  ros::NodeHandle nh;
+  ros::NodeHandle pnh;
+
+  /* Pool of topics subscribers */
+  CmdActuatorSubscribers cmd_fin_angles_subs;
+  CmdActuatorSubscribers cmd_set_rpm_subs;
+
+  /* Multiplexed command Fin Angle topic */
+  ros::Publisher fin_angles_output_topic_pub;
+  ros::Publisher set_rpm_output_topic_pub;
+
+  /* Multiplexed command Fin Angle topic name */
```

Comment:
@patapufate by updating the docblock, I meant to update its content to match what is, not to drop Doxygen syntax. I'm sorry if I wasn't explicit enough.

Reply:
Updated in d64b0a2

Reply:
Updated in 971573c